### PR TITLE
DOC: Add Notes about difference to numpy behaviour for ddof in std() GH35985

### LINKED ISSUE #35985: DOC: make difference between numpy behaviour clearer in Dataframe.std() and Series.std()
#### Location of the documentation

https://pandas.pydata.org/pandas-docs/dev/reference/api/pandas.DataFrame.std.html
https://pandas.pydata.org/pandas-docs/dev/reference/api/pandas.Series.std.html

#### Documentation problem

The `ddof` kwarg has a different default to numpy. This means using the `std()` method on a `Series` or `numpy.array` with the same values give different results:

```python
>>>df=pandas.Series([1,2,3])
>>>df.std()
Out[4]: 1.0
>>>df.values.std()
Out[5]: 0.816496580927726
```

#### Suggested fix for documentation

I assume the difference in behaviour is intentional, but I would suggest making this difference clearer in the documentation.

We could add the text: "Note that this normalization is different to numpy, which by default normalizes by N (equivalent to `ddof=0`)." 
A similar string could be added in the kwarg description for ddof.

This would make it clear from reading the pandas docs that the behaviour is different to numpy, rather than having to compare the numpy docs side by side after noticing a discrepancy. It would also make users more likely to consider which ddof value they want for their use case.

If you agree, I am happy to make the PR to change this.

### PR DESCRIPTION
**Pull request by timhunderwood**
- [x] closes #35985
- [ ] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/core/generic.py [hunk 1]
```diff
@@ -10859,7 +10859,13 @@ def _doc_parms(cls):
...
 Returns
 -------
-%(name1)s or %(name2)s (if level specified)\n"""
+%(name1)s or %(name2)s (if level specified)
+
+Notes
+-----
+Note that the default divisor (normalization) is different to numpy,
+which by default normalizes by N (equivalent to ddof=0). To have the same
+behaviour as numpy, use ddof=0.\n"""
```
Comment (dsaxton) [inline pandas/core/generic.py:10868]:
```suggestion
The default divisor (normalization) is different from numpy, which
by default normalizes by N. To have the same behaviour as numpy,
use ddof=0.\n"""
```

**Review by dsaxton — COMMENTED** [review]
Thanks @timhunderwood, small suggested wording edit

File: pandas/core/generic.py (hunk 1 above)
Comment (timhunderwood) [reply pandas/core/generic.py]: Done @dsaxton , thanks!

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
Thanks @timhunderwood 

> normalizes by N

Are you sure this is correct? Does dividing by N actually normalise the quantity?

IMO it should be enough to write something like
```
Notes
-----
To have the same behaviour as `numpy.std`, use `ddof=0` (instead of the default `ddof=1`)
```

**Comment by timhunderwood** [comment]
> Thanks @timhunderwood
> 
> > normalizes by N
> 
> Are you sure this is correct? Does dividing by N actually normalise the quantity?
> 
> IMO it should be enough to write something like
> 
> ```
> Notes
> -----
> To have the same behaviour as `numpy.std`, use `ddof=0` (instead of the default `ddof=1`)
> ```



> Thanks @timhunderwood
> 
> > normalizes by N
> 
> Are you sure this is correct? Does dividing by N actually normalise the quantity?
> 
> IMO it should be enough to write something like
> 
> ```
> Notes
> -----
> To have the same behaviour as `numpy.std`, use `ddof=0` (instead of the default `ddof=1`)
> ```

thanks @MarcoGorelli , I agree this could be confusing. I've changed to your simpler wording, thanks.

**Review by dsaxton — APPROVED** [review]
Seems like an improvement to me 👍

**Review by MarcoGorelli — APPROVED** [review]
Looks good to me, can you fetch and merge upstream/master and then push again to restart the CI process in order to get it green?

**Comment by timhunderwood** [comment]
> Looks good to me, can you fetch and merge upstream/master and then push again to restart the CI process in order to get it green?

Done - CI process has started...

**Comment by MarcoGorelli** [comment]
Thanks @timhunderwood !

**Pull request merged**